### ISSUE TITLE
Use Java Executable for Windows Binary

### ISSUE THREAD

**Issue by neil-lobo** [pkl-community/node-packages#6]
`Error: Couldn't find application binary inside node_modules for win32-x64`

Currently, Apple only provides binaries for MacOS and Linux in their GitHub release, however, Java executables work for machines that have Java installed

https://pkl-lang.org/main/current/pkl-cli/index.html#java-executable

[Windows support is planned](https://github.com/apple/pkl/issues/20#issuecomment-1925473563) but there is no timeline. If it's possible to add Windows support with the Java executable easily that would be nice :)

**Comment by mattstermiller** [comment]
I found [this unmerged PR commit] with Windows installation notes that was very useful. On top of this I used Chocolatey's shim generator to fake a native exe :)

To generate a shim like I did, fill in the `<ABSOLUTE-PATH-TO-JAR>` in the command below and run it in a Powershell run as Administrator:
```
& "$env:ChocolateyInstall\tools\shimgen.exe" -p "$env:JAVA_HOME\bin\java.exe" -c "-jar <ABSOLUTE-PATH-TO-JAR>\pkl-cli-java.jar" -o "$env:ChocolateyInstall\bin\pkl.exe"
```

This will add the shim to chocolatey's shim folder, but it's up to you to manually update or remove it. And of course, this will break if your current installation of java or the jar is moved/removed.
